tests/unit/data/data_pipeline/test_sample.py

```diff
@@ -10,15 +10,12 @@
 from fairseq2.data import DataPipeline, read_sequence
 from fairseq2.data.text.text_reader import read_text
 from fairseq2.utils.version import is_pt2_or_greater
-from tests.common import python_devel_only, tmp_rng_seed
+from tests.common import tmp_rng_seed
 
 cpu_device = torch.device("cpu")
 
 
-@pytest.mark.skipif(
-    python_devel_only(),
-    reason="New fairseq2n API in Python-only installation. Skipping till v0.2.",
-)
+@pytest.mark.fairseq2n("0.2a0")
```

Comment:
do we need to have a custom pytest marks? 
why not using `skip_if_fairseq2n_newer` as an annotation ?

```
def skip_if_fairseq2n_newer(version):
  ...
  if ...:
     warnings.warn(...)
  return pytest.skip(installed_version < annotated_version, reason=f"The test requires fairseq2n v{annotated_version} or greater.")

```

Reply:
TBH no particular reason. I just followed the instructions here: https://docs.pytest.org/en/7.1.x/example/markers.html. Looks like one advantage of using a pytest marker is that it integrates your marker with the pytest CLI (described [here](https://docs.pytest.org/en/7.1.x/example/markers.html)). It also makes it possible to use a marker in parametrized tests. Let me know if you have any objection on using a pytest marker though. I don't have a strong opinion on this.